Added optional alpha slider, it's set in constructor

konrad-g opened the pull request:
It's a simply update where I added optional alpha slider.

brk3 commented:
Hi, I really just uploaded this mirror to use in one of my projects.

You should probably contact the original author of the code to see if he would like to accept your patch:  http://code.google.com/p/android-color-picker/

If he's not interested or you don't hear back let me know and I can look at merging this.

konrad-g commented:
To be honest I was looking for this repo on GitHub and yours seems to be
most popular. There is no author repo I can improve. I just added feature I
needed and fixed bug occuring on newest API. I want others to benefit from
my work so please review my commits and accept them if you will find them
useful so other developers can also use it.

Regards,
Konrad G
On 28 Jan 2014 12:00, "Paul Bourke" notifications@github.com wrote:

> Hi, I really just uploaded this mirror to use in one of my projects.
> 
> You should probably contact the original author of the code to see if he
> would like to accept your patch:
> http://code.google.com/p/android-color-picker/
> 
> If he's not interested or you don't hear back let me know and I can look
> at merging this.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/brk3/android-color-picker/pull/1#issuecomment-33468664
> .
